proxy/api/src/test_utils.rs

```diff
@@ -0,0 +1,22 @@
+//! Common test utilities
+
+use coco::{
+    identities_payload::{Person, PersonPayload},
+    net::peer::Peer,
+    signer::BoxedSigner,
+    state::{init_owner, Error},
+    LocalIdentity,
+};
+
+/// A facade for [`init_owner`](init_owner)
+/// for creating basic identities without person payload extensions.
+pub async fn init_owner_named(
```

Comment:
Why is was this function introduced, what is the benefit over the normal owner intialisation?

Reply:
It's a test utility. The benefit is that the content of this function would need to be pasted in multiple tests introducing noise.

Reply:
I don't understand how it is different from what `State::init_owner` does under the hood.

Reply:
It's not different, it's a facade, which wraps `state::init_owner`. It hides some features (payload extensions), but offers a cleaner and simpler API, which is good enough for most tests, but requires less clutter.

Reply:
The part I'm confused about is that it's repeated in 3 places now

Reply:
Me too :( We have 3 different test scopes:
1. Coco tests, they run on coco compiled with `cfg(test)`, so they see tools in `mod tests`
2. Api tests, they run on coco compiled as production code, so they can see only the production API. This should be possible to be mitigated with some features and dev-dependencies trickery, but AFAIR you've tried it before and there were some issues. I don't want to bloat the scope of this PR either, I'm fine with adding a `tests` feature and DRY-ing it later.
3. Coco integration tests, they run on coco compiled as production code, so they can see only the production API. AFAIK they have no sane method of accessing the coco test API: https://github.com/rust-lang/cargo/issues/2911

I haven't found any method of DRY-ing this tool without putting it in the production API of coco, where it's unneeded.

Reply:
The problem solved itself, the new production API doesn't need any test tools